remove unused wfp-deploy workflow

olethanh opened the pull request:
Remove the github workflow wfp-deploy as it is outdated and not used anymore since this environment has been removed and deploy method changed

madewulf commented:
Actually, I'm expecting this to happen more in the future. Some forms don't have to be linked to org units

olethanh committed:
remove unused wfp-deploy workflow

Remove the github workflow wfp-deploy as it is outdated and not used anymore
since this environment has been removed and deploy method changed
